### ISSUE TITLE
implement onChanged

### ISSUE THREAD

**Issue by AlexV525** [TinoGuo/pin_input_text_field#17]
既可传controller也可使用onChanged，不强制要求使用controller来监听或控制行为。

```
/// The callback will execute field changed.
final ValueChanged<String> onChanged;
......
/// {@macro flutter.widgets.editableText.onChanged}
///
/// See also:
///
///  * [inputFormatters], which are called before [onChanged]
///    runs and can validate and change ("format") the input value.
///  * [onEditingComplete], [onSubmitted], [onSelectionChanged]:
///    which are more specialized input change notifications.
final ValueChanged<String> onChanged;
```

**Comment by TinoGuo** [comment]
It will be implemented in the next version.

**Comment by AlexV525** [comment]
Thanks. It helps a lot 👍

**Comment by TinoGuo** [comment]
implemented after bf7d1e3
